### DISCUSSION TITLE
LLM bahasa

### DISCUSSION THREAD

**Discussion by naturallyfunny** [teamalgoritma/community#872] [Q&A] [answered]
Model conversational LLM bahasa di hugging face yang bagus yang mana ya?

**Comment by saskia-dwi-ulfah** [comment, marked as answer]
Dear @ardianthescientist 

Berikut leaderboard LLM di HuggingFace yang bisa dicoba:
https://huggingface.co/collections/open-llm-leaderboard/llm-leaderboard-best-models-652d6c7965a4619fb5c27a03

Semoga jawaban ini dapat membantu.

Best regards,
Team Algoritma

**Comment by ramdhinafinita** [comment]
Halo @ardianthescientist, untuk menjaga agar forum diskusi berjalan kondusif, karena tidak ada respon lanjutan setelah 7 hari dari respon terakhir, kami izin menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.
